packages/viewer/src/interactions/interactionApi.ts

```diff
@@ -191,6 +195,33 @@ export class InteractionApi {
     return this.currentCamera != null;
   }
 
+  /**
+   * Scales the provided `pixelThreshold` by the device's pixel ratio
+   * and the configured `coarsePointerThresholdScale` value if the
+   * primary pointer input is coarse (touch).
+   *
+   * Scales the provided `pixelThreshold` by the configured
+   * `finePointerThresholdScale` value if the primary pointer input is
+   * fine (mouse).
+   *
+   * @param pixelThreshold - The pixel threshold to scale.
+   * @returns The scaled pixel threshold.
+   */
+  public scalePixelThreshold(
+    pixelThreshold: number,
+    isTouch?: boolean
+  ): number {
+    if (this.isCoarseInputDevice(isTouch)) {
+      return (
+        pixelThreshold *
+        window.devicePixelRatio *
+        this.getConfig().coarsePointerThresholdScale
```

Comment:
Thoughts on simplifying this to simply `course|fineThreshold * window.devicePixelRatio`? I think this additional scaling factor may be more difficult for external devs to understand vs a simpler setting that is just `pixel threshold * DPI`.

Reply:
Yeah, I think that makes sense. I suppose in the case of a higher `window.devicePixelRatio` on a laptop/desktop, we still may want to increase the threshold, so I can update this